ui/components/src/TimeSeriesTooltip/focused-series.ts

```diff
@@ -144,15 +190,8 @@ export function getFocusedSeriesData(
   if (chart['_model'] === undefined) return [];
   const chartModel = chart['_model'];
   const yAxisInterval = chartModel.getComponent('yAxis').axis.scale._interval;
-
-  const seriesNum = chartData.timeSeries.length;
-
-  // tooltip trigger area gets smaller with more series, increase yAxisInterval multiplier to expand nearby series range
-  const yBuffer =
-    seriesNum > SHOW_FEWER_SERIES_LIMIT
-      ? yAxisInterval * REDUCE_FOCUSED_SERIES_MULTIPLIER
-      : yAxisInterval * INCREASE_FOCUSED_SERIES_MULTIPLIER;
-
+  const totalSeries = chartData.timeSeries.length;
+  const yBuffer = getYBuffer({ interval: yAxisInterval, totalSeries, showAllSeries });
```

Comment:
I'm not opposed to calling the argument `yAxisInterval` or `yInterval` (just to make the connection to the y-axis obvious, I can see myself wondering what `interval` means by itself)

up to you!